main.go

```diff
@@ -92,7 +92,13 @@ func main() {
 	if kubeConfig.UserAgent == "" {
 		kubeConfig.UserAgent = useragent.Default()
 	}
-
+	if cfg.Client != nil && cfg.Client.QPS != nil {
```

Comment:
we set these defaults in apis/config/v1alpha2/defaults.go

Reply:
Do you mean that if QPS is not set we should default it to controller-runtime default of 20?

Reply:
no, I mean that this is not the correct place in the codebase to put the defaults. It should be in the apis/config/v1alpha2/defaults.go file.